### PR TITLE
ARUHA-906: Move nakadi-manual to nakadi/docs

### PR DESCRIPTION
**Pull request by SergKam**
ARUHA-906: Merge Documentation
https://techjira.zalando.net/browse/ARUHA-906

## Description
As we have too many sources of information for users, they not always in sync.
To make it easy we decided to merge documentation from nakadi-manual to nakadi/docs
Also, it allows us to use github-pages naturally to build whole web site for Nakadi project
and keep Open API and manual in sync automatically.

## Review
- [ ] Documentation
- [x] CHANGELOG

### CODE REVIEW

File: docs/_documentation/intro.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+HTTP/1.1 200 OK
+
+{"cursor":{"partition":"0","offset":"4"},"events":[{"order_number": "ORDER_001", "metadata": {"eid": "4ae5011e-eb01-11e5-8b4a-1c6f65464fc6", "occurred_at": "2016-03-15T23:56:11+01:00"}}]}
+{"cursor":{"partition":"0","offset":"5"},"events":[{"order_number": "ORDER_002", "metadata": {"eid": "4bea74a4-eb01-11e5-9efa-1c6f65464fc6", "occurred_at": "2016-03-15T23:57:15+01:00"}}]}
+{"cursor":{"partition":"0","offset":"6"},"events":[{"order_number": "ORDER_003", "metadata": {"eid": "4cc6d2f0-eb01-11e5-b606-1c6f65464fc6", "occurred_at": "2016-03-15T23:58:15+01:00"}}]}
+```
+
+## Nakadi community
+
+Nakadi ecosystem has a lot of related projects. Please check [zalando-nakadi](https://github.com/zalando-nakadi/)
```
Comment (lmontrieux) [inline docs/_documentation/intro.md]: `Nakadi ecosystem has a lot of related projects. Please check` -> `There is a large ecosystem of projects around Nakadi. Check they out on`

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            </li>
+        </ul>
+    </nav>
+</header>
+
+<section class="banner">
+    <div class="column">
+        <h1>Nakadi</h1>
+        <h3>Event Bus Broker</h3>
+        <p>A distributed event bus that implements a RESTful API abstraction instead of Kafka-like queues</p>
```
Comment (lmontrieux) [inline docs/index.html]: `instead of` -> `on top of`

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    </article>
+    <article><h1>JSON Schema</h1>
+        <p>Enable convenient development of event-driven applications and asynchronous microservices. Event types can be
+            defined with schemas and managed via a registry. Nakadi also has optional support for events describing
+            business processes and data changes using standard primitives for identity, timestamps, event types, and
+            causality.</p>
+    </article>
+    <article><h1>Flexibility</h1>
+        <p>Nakadi instances are stateless. They can be run on AWS with auto-scaling.</p>
+        <p>Using Timelines it is easy to move the traffic to another cluster without moving the data and any service
```
Comment (lmontrieux) [inline docs/index.html]: `Using Timelines it is easy` -> `Using Timelines, it is easy`
`without moving the data and any service degradation` -> `without moving the data, and without service interruption`.

Also, this is the first mention of timelines, and this does not explain what they are.

**Comment by lmontrieux** [comment]
@SergKam I would also like to have some documentation on how to build the pages locally

**Comment by lmontrieux** [comment]
👍

**Comment by SergKam** [comment]
:+1:

**Pull request merged**